src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SFA.DAS.FAA.Domain.Interfaces;
+
+namespace SFA.DAS.FAA.Domain.Applications.GetSubmittedApplications
+{
+    public record GetSubmittedApplicationsApiRequest(Guid CandidateId) : IGetApiRequest
+    {
+        public string GetUrl => $"applications/submitted?candidateId={CandidateId}";
```

Comment:
I feel like the route for this should be more like:
```
users/account-deletion
```
so it matches whats happening in the UsersController

Reply:
Route path changed in APIM & web